# COEX-seq
COEX-seq: COnvert a variety of measurements of gene EXpression in RNA-seq

Sang Cheol Kim, Seong Beom Cho

Division of Biomedical Informatics, Center for Genome Science, National Institute of Health, KCDC, Choongchung-Buk-do 363-951, Republic of Korea

Next-generation sequencing(NGS), a well-known technology, is being affected by the trends in transcriptome studies. Many types of transcriptome studies, such as mRNA and miRNA, have been widely studied through quantitative analysis. In recent years, most studies of the transcriptome have been done by RNA-seq experiment. Many methods have been developed to solve different aspects of RNA-seq data analysis, but combining them into a robust analysis pipeline is an ongoing challenge. In particular, databases such as GEO(Gene Expression Omnibus), ArrayEXpress (https://www.ebi.ac.uk/arrayexpress/), and TCGA(The Cancer Genome Atlas)  provide the RNA-seq data in non-unified formats. Utilizing different quantitative measurements provided by public databases is limited to integration analysis. To solve this problem, the COEX-seq system was developed to easily convert data into a variety of formats. COEX-seq is a web application(Shiny; a web application framework for R) framework for converting a variety of measurements of gene expression in RNA-seq experiments. It is provides a workflow that includes data loading, choosing measurements of gene expression, and selection and identification of gene names. COEX-seq is freely available for academic purposes and can be run on Windows, Mac, and Linux operating systems. Source code, sample data sets, and supplementary documentation can be found at https://github.com/bioinfo-korea/COEX-seq.git.

Keywords: RNA-seq, measurements of gene expression, a web application framework for R, integrated analysis
